File: Exercicios-Beecrowd/Python/1045.py
a,b,c=input().split()
a=float(a)
b=float(b)
c=float(c)
lista=[]
lista.append(a)
lista.append(b)
lista.append(c)
lista.sort()
if (lista[2]**2)>(lista[0]**2)+(lista[1]**2):
    if lista[2]>=lista[0]+lista[1]:
        print('NAO FORMA TRIANGULO')
    elif lista[2]==lista[0] or lista[0]==lista[1] or lista[1]==lista[2]:
        print('TRIANGULO OBTUSANGULO')
        print('TRIANGULO ISOSCELES')
    else:
        print('TRIANGULO OBTUSANGULO')
elif (lista[2]**2)==(lista[0]**2)+(lista[1]**2):
    print('TRIANGULO RETANGULO')
elif (lista[2]**2)<(lista[0]**2)+(lista[1]**2):
    print('TRIANGULO ACUTANGULO')
    if lista[2]==lista[0]==lista[1]:
        print('TRIANGULO EQUILATERO')
    elif lista[2]==lista[0] or lista[0]==lista[1] or lista[1]==lista[2]:
        print('TRIANGULO ISOSCELES')

File: Exercicios-Beecrowd/Kotlin/1045.kt
import java.util.*
import kotlin.math.pow

fun main(args: Array<String>) {

    val num= readLine().toString().split(" ")
    var lista= arrayOf<Double>()
    for (x in num){
        lista+=x.toDouble()
    }
    lista.sort()
    val A=lista[2]
    val B=lista[1]
    val C=lista[0]






    if (A >= B+C){
       println("NAO FORMA TRIANGULO")
   }
    else if(A.pow(2) == (B.pow(2) + C.pow(2))){
        println("TRIANGULO RETANGULO")

    }
    else if (A.pow(2) > ((B.pow(2) + C.pow(2)))){
        println("TRIANGULO OBTUSANGULO")
   }
    else if (A.pow(2) < ((B.pow(2) + C.pow(2)))){
        println("TRIANGULO ACUTANGULO")

   }
    if (A==B && B==C){
        println("TRIANGULO EQUILATERO")
    }
    else if(A==B || B==C || C==A) {
        println("TRIANGULO ISOSCELES")

    }
	
}